README.md

```diff
@@ -1,70 +1,69 @@
-# Getting Started with Create React App
+# Country-Facts
+Contries facts is a a React and Redux Asing Page Application(SPA) application that dispalys key information about various countries in the world. 
+## Built With the following
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+- CSS
+- React.js
+- Javascript
+- Babel
+- Redux
+- Linters
 
-## Available Scripts
+## Live Demo
+[Live Demo Link](https://benardop.github.io/country-facts/)
```

Comment:
- [ ] Kindly ensure your app is properly deployed. When I visit the site, it displays nothing. Kindly fix that.

  ![fkkgi](https://user-images.githubusercontent.com/36542986/195560693-57b8f0f0-4770-40f9-a30b-8453906c6343.jpg)